hwdb.d/60-sensor.hwdb

```diff
@@ -675,6 +675,11 @@ sensor:modalias:acpi:SMO8500*:dmi:bvnAmericanMegatrendsInc.:bvr5.6.5:bd07/25/201
 # One-netbook
 #########################################
 
+# One-netbook OneMix 2s
+# OneMix 2s has no product name filled, matching entire dmi-alias
+sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvr5.12:bd10/26/2018:br5.12:svnDefaultstring:pnDefaultstring:pvrDefaultstring:rvnDefaultstring:rnDefaultstring:rvrDefaultstring:cvnDefaultstring:ct3:cvrDefaultstring:*
```

Comment:
Not sure, but looks dangerous...

Reply:
I was using [this](https://github.com/systemd/systemd/pull/19549) as guideline and wanted to share my accel matrix with other OneMix 2s users.

Reply:
Oh, we already have similar lines. hmm. ok.